src/Workspaces/Core/Portable/Remote/RemoteServiceName.cs

```diff
@@ -61,6 +61,8 @@ public string ToString(bool isRemoteHost64Bit)
                 (WellKnownServiceHubService.RemoteProjectTelemetryService, true) => Prefix + nameof(WellKnownServiceHubService.RemoteProjectTelemetryService) + Suffix64,
                 (WellKnownServiceHubService.RemoteTodoCommentsService, false) => Prefix + nameof(WellKnownServiceHubService.RemoteTodoCommentsService),
                 (WellKnownServiceHubService.RemoteTodoCommentsService, true) => Prefix + nameof(WellKnownServiceHubService.RemoteTodoCommentsService) + Suffix64,
+                (WellKnownServiceHubService.RemoteEditAndContinueService, false) => Prefix + nameof(WellKnownServiceHubService.RemoteEditAndContinueService),
+                (WellKnownServiceHubService.RemoteEditAndContinueService, true) => Prefix + nameof(WellKnownServiceHubService.RemoteEditAndContinueService) + Suffix64,
```

Comment:
bte, this whole switch would be simpler if this whole switch ignored the 64bit case, and just added Suffix64 unilaterally at the end.